src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.ManagedSslBio.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Runtime.InteropServices;
+using Microsoft.Win32.SafeHandles;
+using System.Diagnostics;
+
+internal static partial class Interop
+{
+    internal static partial class Crypto
+    {
+        internal static class ManagedSslBio
+        {
+            private unsafe static ReadDelegate s_readDelegate;
+            private unsafe static WriteDelegate s_writeDelegate;
+
+            static ManagedSslBio() => Initialize();
+
+            internal static SafeBioHandle CreateManagedSslBio() => Crypto.CreateManagedSslBio();
+            
+            private unsafe static void Initialize()
+            {
+                s_writeDelegate = Write;
+                s_readDelegate = Read;
+                Crypto.InitManagedSslBioMethod(s_writeDelegate, s_readDelegate);
```

Comment:
For `Interop.Crypto` it's broken up like that so it can be cascaded by `Interop.Http`